accelerator/message.h

```diff
@@ -6,13 +6,10 @@ extern "C" {
 #endif
 
 typedef enum ta_cli_arg_value_e {
-  HELP_CLI,
-
   /** TA */
   TA_HOST_CLI,
```

Comment:
set `TA_HOST_CLI = 127` or even bigger number, since the enum may have a confict with the ascii `int val` field of `ta_cli_arguments_g` in the future